apps/dashboard/app/models/workflow.rb

```diff
@@ -89,8 +93,51 @@ def manifest_file
     Workflow.workflow_dir(@project_dir).join("#{@id}.yml")
   end
 
-  # TODO: Add logic to save the DAG relation between launchers like array of <launcher #1, launcher #2>
+  def submit(attributes = {})
```

Comment:
How are you able to test this given you can't save the workflow?

Reply:
<img width="996" height="673" alt="image" src="https://github.com/user-attachments/assets/b81fdf90-bab1-4015-afc3-89b5c8ed815a" />

So all scripts have a sleep delay, I check when is which one launched to see if order is as expected

<img width="330" height="354" alt="image" src="https://github.com/user-attachments/assets/7e1cc10d-73bc-44a1-865b-44788cbfb721" />